ateam_common/include/ateam_common/udp_sender.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 A Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#ifndef ATEAM_COMMON__UDP_SENDER_HPP_
+#define ATEAM_COMMON__UDP_SENDER_HPP_
+
+#include <boost/asio.hpp>
+
+#include <functional>
+#include <string>
+
+namespace ateam_common
+{
+class UDPSender
+{
+public:
+  UDPSender(
+    std::string udp_ip_address,
+    int16_t udp_port);
```

Comment:
These should both be read-only.

```suggestion
  UDPSender(
    const std::string& udp_ip_address,
    const int16_t udp_port);
```